controllers/admin/AdminImportController.php

```diff
@@ -1260,161 +1805,180 @@ public function categoryImport($offset = false, $limit = false, &$crossStepsVari
+                    if ($fieldError !== true || isset($langFieldError) && $langFieldError !== true) {
+                        $this->errors[] = ($fieldError !== true ? $fieldError : '')
+                            . (isset($langFieldError) && $langFieldError !== true ? $langFieldError : '')
+                            . Db::getInstance()->getMsgError();
                     }
                 }
             }
         }
-        if (isset($category->link_rewrite) && !empty($category->link_rewrite[$default_language_id])) {
-            $valid_link = Validate::isLinkRewrite($category->link_rewrite[$default_language_id]);
+        if (isset($category->link_rewrite) && !empty($category->link_rewrite[$defaultLanguageId])) {
+            $validLink = Validate::isLinkRewrite($category->link_rewrite[$defaultLanguageId]);
         } else {
-            $valid_link = false;
+            $validLink = false;
         }
 
-        if (!$shop_is_feature_active) {
+        if (!$shopIsFeatureActive) {
             $category->id_shop_default = 1;
         } else {
             $category->id_shop_default = (int)Context::getContext()->shop->id;
         }
 
-        $bak = $category->link_rewrite[$default_language_id];
-        if ((isset($category->link_rewrite) && empty($category->link_rewrite[$default_language_id])) || !$valid_link) {
-            $category->link_rewrite = Tools::link_rewrite($category->name[$default_language_id]);
+        $bak = $category->link_rewrite[$defaultLanguageId];
+        if ((isset($category->link_rewrite) && empty($category->link_rewrite[$defaultLanguageId])) || !$validLink) {
+            $category->link_rewrite = Tools::link_rewrite($category->name[$defaultLanguageId]);
             if ($category->link_rewrite == '') {
                 $category->link_rewrite = 'friendly-url-autogeneration-failed';
-                $this->warnings[] = sprintf($this->trans('URL rewriting failed to auto-generate a friendly URL for: %s', array(), 'Admin.Advparameters.Notification'), $category->name[$default_language_id]);
+                $this->warnings[] = $this->trans('URL rewriting failed to auto-generate a friendly URL for: %s',
+                    array($category->name[$defaultLanguageId]),
+                    'Admin.Advparameters.Notification'
+                );
             }
             $category->link_rewrite = AdminImportController::createMultiLangField($category->link_rewrite);
         }
 
-        if (!$valid_link) {
+        if (!$validLink) {
             $this->informations[] = $this->trans('Rewrite link for %1$s (ID %2$s): re-written as %3$s.', array(
                 '%1$s' => $bak,
-                '%2$s' => (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null',
-                '%3$s' => $category->link_rewrite[$default_language_id],
+                '%2$s' => $categoryId ? $categoryId : 'null',
+                '%3$s' => $category->link_rewrite[$defaultLanguageId],
```

Comment:
This is the perfect moment to update the variable names to make them more explicit. If needed you can check with @AlexEven